packages/core/admin/admin/src/content-manager/components/DynamicZone/components/DynamicComponent.js

```diff
@@ -180,11 +133,70 @@ const DynamicZoneComponent = ({
       >
         <Drag />
       </IconButton>
+      <Menu.Root>
+        <Menu.Trigger size="S" endIcon={undefined} paddingLeft={2} paddingRight={2}>
+          <More aria-hidden focusable={false} />
+          <VisuallyHidden as="span">
+            {formatMessage({
+              id: getTrad('components.DynamicZone.more-actions'),
+              defaultMessage: 'More actions',
+            })}
+          </VisuallyHidden>
+        </Menu.Trigger>
+        <Menu.Content>
+          <Menu.SubRoot>
+            <Menu.SubTrigger>
+              {formatMessage({
+                id: getTrad('components.DynamicZone.add-item-above'),
+                defaultMessage: 'Add item above',
```

Comment:
Shouldn't we say "component" instead of "item"?

Reply:
We totally could, I personally found the length of the label too long, @maevalienard what do you think would be best? :)

Reply:
I think "component" is fine, I prefer to make a compromise on the length rather than on the specificity :)

Reply:
I'll make this change